azure-pipelines.yml

```diff
@@ -62,11 +62,11 @@ jobs:
       # https://developercommunity.visualstudio.com/content/problem/689794/pipelines-logging-command-logissue-does-not-report.html
       #echo "##vso[task.issue type=error;sourcepath=/azure-pipelines.yml;linenumber=45;columnnumber=1;code=100;]Found something that could be a problem."
       fork_origin=$(git merge-base --fork-point origin/master)
-      changed_files=$(git diff --name-only $fork_origin | grep -v /vendor/)
+      changed_files=$(git diff --name-only $fork_origin | grep -v /vendor/ | grep -E '\.(cpp|cc|c|h)$')
       test -z $changed_files || git diff -U0 $fork_origin $changed_files | clang-format-diff -p1 | tee clang-format-output
       if [ -s clang-format-output ]; then
         echo -n "##vso[task.logissue type=error]"
-        echo "C/C++ lint failed. Use 'util/run-clang-format.sh' or 'git clang-format' to format the code."
+        echo "C/C++ lint failed. Use 'git clang-format' to reformat the changed code."
```

Comment:
`git clang-format` on its own will only work for staged changes? This advice could then cause confusion if one does not know how to identify the conflicting commit and get the changes into the staging area.

Reply:
How about "Use `git clang-format` with appropriate options to reformat the changed code." In a short message, we can't really give universal instructions but you're right that the current message implies that just using `git clang-format` without any options will be sufficient.

Reply:
I agree that extensive make no sense here. Your suggestion make sense, so feel free to update.

Reply:
I made that change, thanks for the suggestion to tweak this.